index.html

```diff
@@ -11,23 +11,21 @@
     <script src="js/d3.min.js"></script>
     <!-- some utility functions that are highly useful -->
     <script src="/networktables/utils.js"></script>
-    <script src="js/dashboard.js"></script>
     <script src="/networktables/camera.js"></script>
+    <script src="js/dashboard.js"></script>
 </head>
 
 <body>
 
-    <div class="topPanel">
-        <p class="timer" id="cycleTimer">000</p>
+    <div id="top-pane">
+        <span class="timer" id="cycleTimer">000</span>
         <select id="auto-select"></select>
         <span id="connection"></span>
     </div>
 
-    <div id="camera" class="camera inital-hide"></div>
-
-    <form class="checklist-div" id="checklist">
+    <form id="checklist">
         <label><input id="check-connection" type="checkbox"> Is robot connected?<br></label>
-        <label><input id="auto" type="checkbox"> Select Auto<br></label>
+        <label><input id="check-auto" type="checkbox"> Select Auto<br></label>
         <label><input type="checkbox"> Is cube loaded?<br></label>
         <label><input type="checkbox"> Is controller connected and in correct order.?<br></label>
```

Comment:
Also this should be changed to Is controller connected and in correct order?

Reply:
Well I never touched now, did I.